src/Catty/DataModel/VariablesContainer/UserDataContainer.h

```diff
@@ -31,29 +31,20 @@
 
 @interface UserDataContainer : NSObject
 
-// Map<Sprite, List<UserVariable>
-@property (nonatomic, strong) OrderedMapTable *objectListOfLists;
-@property (nonatomic, strong) OrderedMapTable *objectVariableList;
+- (NSMutableArray<UserList*> *)lists;
```

Comment:
Please change to _NSArray_ or similar so that it is forbidden to add new elements to the array from outside.